deployment/kubernetes/Chart.yaml

```diff
@@ -5,9 +5,9 @@ version: "1.0.0"
 # The appVersion defines which docker image is pulled.
 # Having it set to latest will pull the latest build on dockerhub.
 # You are free to define a specific version here tho.
-# e.g. appVersion: "0.6.10"
-# Be aware that this requires all dockers to have the same version available.
-appVersion: "latest"
+# e.g. appVersion: "1.0.2-79"
+# Be aware that this requires all your apps to have the same docker image version available.
+appVersion: "1.0.2-79"
```

Comment:
i think here we can keep `latest` since we pull the images from `"${DOCKER_ORGANISATION_PUSH}/webapp-branded` which should be fine to use latest.

Reply:
Yes. I agree.